chart/templates/server-deployment.yaml

```diff
@@ -188,6 +188,8 @@ spec:
               key: auth-providers.json
         - name: BRANDING_CONFIG
           value: {{ .Values.branding | toJson | quote }}
+        - name: ADMIN_LINKS
```

Comment:
AFAIK the plan was to make those links a "com" concern rather  than introduce generic means here.
What made you change your mind?

Reply:
> What made you change your mind?

Nothing, I just followed the pattern here: Admin interfaces and extension points are implemented in `core`, actual implementation and usage in `com`.

Reply:
Some details we can discuss and switch easily, e.g. move the dashboard part from `com` to `ee`, which would make total sense.

But breaking the existing pattern by introducing even more hinges is not worth the cost IMO. But let's discuss in sync.

Reply:
I wasn't asking to break patterns. One pattern that we have is that in `com` we can add additional properties, like those links.

I really don't think we need to configurable links here, but this is a com only concern.
We need the API extension in ws-manager to get the data - I don't think we need to add/maintain configurable links here.